test/02_ostprime/01_setup_ost.js

```diff
@@ -1,13 +1,14 @@
 const EIP20 = require('../../src/eip20');
 const shared = require('../shared');
 const Deployer = require('../../src/deployer');
+const config = require('../config');
 
 describe('OST Prime setup', () => {
   it('deploy eip20 token', async () => {
```

Comment:
Move `const decimal = config.decimals;` out of `it` in `describe`
```suggestion
it(`deploy eip20 token (decimals: ${decimal})`, async () => {
```